scripts/check-sample-lnd-conf.sh

```diff
@@ -14,33 +14,35 @@ CONF_FILE=${2:-sample-lnd.conf}
 # We are reading the default values of lnd from lnd --help. To avoid formatting
 # issues the width of the terminal is set to 240. This needs a workaround for
 # CI where we don't have an interactive terminal.
+FILE_TMP=$(mktemp)
 if [ -t 0 ]; then
     stty cols 240
-    LND_HELP="$(go run -tags="$TAGS" \
-        github.com/lightningnetwork/lnd/cmd/lnd --help)"" --end"
+    go run -tags="$TAGS" github.com/lightningnetwork/lnd/cmd/lnd --help > \
```

Comment:
One thing I noticed while testing is that after I run the script locally, my shell continues using the 240 size. Maybe we can restore the old value?
We can find out the current value with `size=($(stty size))` and then restore it with `stty cols $size[2]`.

Reply:
Added, but restore with `stty cols ${size[1]}`

Reply:
Cool! Yeah, not sure why it was index 2 for me, probably just because of the way I invoked it. Can confirm this fix works.